scripts/get_max_api_locators.sh

```diff
@@ -23,7 +23,7 @@ function main() {
 
         local largestLocator=$(grep -R '("-' "${path}" | sed 's/^.*"\(-[0-9]\{3\}\)".*$/\1/' | sort | uniq | tail -n 1)
         # Remove input zero padding (so bash does not think the numebr is octal.
-        local cleanLargestLocator=$(echo "${largestLocator}" | sed 's/-0\+/-/g')
+        local cleanLargestLocator=$(echo "${largestLocator}" | sed 's/^-0*/-/g')
```

Comment:
The anchor makes sense, but I think plus makes sense here more than star.
Because of the previous line they will both work the same,
but I think the star more clearly shows our intentions.
One of the harder things to really wrap your head around when writing maintainable code is creating code that not only correctly does what you want, but that also expresses your intention.

Reply:
After playing around with it more, it turns out the only thing both sed's needed was a -E flag. Should I still include the anchor even though it works without it now? Also, did you mean to say the plus more clearly shows our intentions?